### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by ghostant-1017**

### CODE REVIEW

**Review by gluax — DISMISSED** [review]

File: synthesizer/src/vm/helpers/rewards.rs [hunk 1]
```diff
@@ -34,7 +34,7 @@ const MAX_COINBASE_REWARD: u64 = ledger_block::MAX_COINBASE_REWARD; // Coinbase
 ///
 /// This method ensures that stakers who are bonded to validators with more than **25%**
 /// of the total stake will not receive a staking reward. In addition, this method
-/// ensures stakers who have less than 10 credit are not eligible for a staking reward.
+/// ensures stakers who have less than 10,000 credit are not eligible for a staking reward.
```
Comment (evanmarshall) [inline synthesizer/src/vm/helpers/rewards.rs]:
```suggestion
/// ensures delegators who have less than 10,000 credits are not eligible for a staking reward.
```

**Review by evanmarshall — CHANGES_REQUESTED** [review]
One more typo to fix. I added the suggestion as a comment.

**Review by Meshiest — DISMISSED** [review]

**Comment by ghostant-1017** [comment]
> One more typo to fix. I added the suggestion as a comment.

fixed

**Review by evanmarshall — APPROVED** [review]
lgtm :shipit:

**Pull request merged**